github-management/OWNERS

```diff
@@ -1,13 +1,17 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+  - cblecker
+  - spiffxp
+
 reviewers:
   - calebamiles
-  - cblecker
   - fejta
-  - grodrigues3
   - idvoretskyi
-  - spiffxp
-approvers:
-  - cblecker
-  - grodrigues3
+  - justaugustus
+  - mrbobbytables
```

Comment:
Was there a comment about this?  I didn't see it elsewhere

Reply:
>the membership team are also listed as reviewers here

Reply:
To unpack that a bit, @mrbobbytables and @justaugustus are already doing a bunch of work related to org membership (listed later in the README.md)

The docs describing what they do live here, and I want to make sure they get suggested as reviewers

Reply:
Ok